train.ipynb

```diff
@@ -1,247 +1,249 @@
-   },
-   "outputs": [],
-   "source": [
-    "!pip install -r requirements.txt"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "colab_type": "text",
-    "id": "2HichwT2JR4W"
-   },
-   "source": [
-    "*Run the model*"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 0,
-   "metadata": {
-    "colab": {
-     "autoexec": {
-      "startup": false,
-      "wait_interval": 0
-     }
+    {
+      "metadata": {
+        "id": "nJ31i4Z8JHa8",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Cloning the repository*"
+      ]
     },
-    "colab_type": "code",
-    "id": "K8peoV51Fj4A"
-   },
-   "outputs": [],
-   "source": [
-    "!python3 train.py -n=inception -o=adam -ct='Y vs D + N'"
-   ]
-  }
- ],
- "metadata": {
-  "colab": {
-   "default_view": {},
-   "name": "load_repo.ipynb",
-   "provenance": [],
-   "version": "0.3.2",
-   "views": {}
-  },
-  "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.5.5"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 1
-}
+    {
+      "metadata": {
+        "id": "NPOQ5rFHFbnU",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!git lfs clone --recurse-submodules https://github.com/marco-c/autowebcompat.git"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "8E5K5gxGJKrW",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Marking working directory*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "GIxBvGGdFdoK",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "os.chdir('autowebcompat')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "ugZB2EhbJOui",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "*Installing requirements*"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "wBqw99O-FfTK",
+        "colab_type": "code",
+        "colab": {
+          "autoexec": {
+            "startup": false,
+            "wait_interval": 0
+          }
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "%%writefile requirements.txt\n",
+        "selenium==3.4.3\n"
```

Comment:
So Keras and Tensorflow are already available?
Since `selenium` is the only needed package, you could just do `!pip install selenium==3.4.3`.

Reply:
Yeah they are already available. Pre-installed for GPU version. I was just overwriting them with CPU version :disappointed: .  We need to install selenium only.